app/views/feed/contentSelectorPreview.phtml

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html>
+	<body>
+		<?php
+			if ($this->fatalError != '') {
+		?>
+		<p class="warning"><?= $this->fatalError ?></p>
+		<?php
+			} else {
+				if ($this->selectorSuccess === false) {
+		?>
+		<p class="warning"><?= _t('feedback.sub.feed.selector_preview.no_result') ?></p>
+		<hr>
+		<?php
+				}
+		?>
+		<center>
+			<div>
+				<input type="checkbox" id="freshrss_rendered" name="freshrss_rendered" checked>
+				<label for="freshrss_rendered"><?= _t('sub.feed.selector_preview.show_rendered') ?></label>
+
+				<input type="checkbox" id="freshrss_raw" name="freshrss_raw">
+				<label for="freshrss_raw"><?= _t('sub.feed.selector_preview.show_raw') ?></label>
```

Comment:
I would use a radio button to switch between the two views. Maybe not perfect either, but the problem of checkboxes is:

- if you uncheck both, the screen is empty
- if both are checked, the source code is not immediately visible and user might think it doesn't work

Reply:
Yes, actually it's what I thought about the two box unchecked. I wanted to add some code to force to have at least one box checked, but laziness…

And I agree this UI is a bit raw. I'm not skilled enough to do something better (so my comment about your proposal to take care of this after merging).

Fixed.

Reply:
Please never write code to make a checkbox behave like a radiobox. ;-)

Reply:
Well, I agree, but it was not exactly the behavior of radiobox, as it would have been possible to select the two box at the same time. The idea was just to make impossible to deselect the two altogether.

Reply:
Oh, just add a `disabled` attribute to the one. Just a couple lines of JS. :-P